public/collections.json

```diff
@@ -64,5 +64,24 @@
       "en": "Online dating",
       "fr": "Rencontre en ligne"
     }
+  },
+  "Demo": {
+    "languages": ["en"],
+    "jurisdictions": ["EU"],
+    "maintainers": [
+      {
+        "name": "Open Terms Archive",
+        "logo": "/images/logo/logo-open-terms-archive-black.png",
+        "url": "https://opentermsarchive.org"
+      }
+    ],
+    "stats": {
+      "services": "2",
```

Comment:
I thought we also had SendInBlue, but I guess I missed something 😄

Reply:
From what I remember SendInBlue was discarded because it is not a necessary service for the engine to work.